spec/requests/attributes/names_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Attributes::NamesController do
+  before do
+    stub_oidc_discovery
+
+    fixture_file = YAML.safe_load(File.read(Rails.root.join("spec/fixtures/user_attributes.yml"))).with_indifferent_access
+    allow(UserAttributes).to receive(:load_config_file).and_return(fixture_file)
+  end
+
+  let(:session_identifier) { placeholder_govuk_account_session }
+  let(:headers) { { "Content-Type" => "application/json", "GOVUK-Account-Session" => session_identifier } }
+
+  # names must be defined in spec/fixtures/user_attributes.yml
+  let(:attribute_name1) { "test_attribute_1" }
+  let(:attribute_value1) { "some_value" }
+
+  let(:params) { { attributes: [attribute_name1] } }
+  let(:status) { 200 }
+
+  describe "GET #show" do
```

Comment:
Could we also have a test here for requesting two attributes, where one of them doesn't have a value?

Reply:
Definitely! That's a good idea.